client/state/data-layer/wpcom/account-recovery/index.js

```diff
@@ -10,4 +10,11 @@ import requestReset from './request-reset';
 import reset from './reset';
 import validate from './validate';
 
-export default mergeHandlers( lookup, requestReset, reset, validate );
+import { registerHandlers } from 'state/data-layer/handler-registry';
+
+registerHandlers(
+	'state/data-layer/wpcom/account-recovery/index.js',
```

Comment:
Do we need some kind of convention on how we name registered handlers? [In the original PR](https://github.com/Automattic/wp-calypso/pull/25073/files) the `/index.js` was omitted for IDs which used a path-like format. There's also some `registerHandler` calls which only use a single keyword.

Reply:
the top-level ones were a compromise to remove the top-level `mergeHandlers` but all others I felt deserve the full path. I'm not opposed to putting `client` in there but meh, no deep need.

we spoke about this on Slack.

the most important thing to me is maintaining a uniqueness constraint and file paths work. originally I tried using `__filename` but that wasn't replace. it's possible we could use have webpack replace it with the real filename but in the end I didn't feel it was worth it for this work.

do we need the id? I don't know. if we ever want to remove handlers or prevent double-insertion then yeah we need it.

Reply:
Agreed. I think having `registerHandlers` warn us in dev about possible duplications is sufficient.